packages/amplication-e2e-tester/Login test/login.test.js

```diff
@@ -0,0 +1,17 @@
+describe('login test', () => {
+  beforeAll(async () => {
+    await page.goto('http://localhost:3001/login');
+  },60000); 
+  it('should log into user account ', async () => {
+    page.setDefaultTimeout(60000)

```

Comment:
Extract to a constant at the beginning of the file. Constants should be in UPPER_CASE format.